cmd/ltkeycloak/README.md

```diff
@@ -24,6 +28,7 @@ Global Flags:
       --dry-run                  perform a dry run without making any changes
       --keycloak-host string     keycloak host (default "http://localhost:8484")
       --mattermost-host string   The Mattermost host to migrate users from
+      --migrate-all-users        Migrate all users ignoring their current auth method
```

Comment:
Maybe `--force-migrate` sounds better given that we are ignoring the current auth method? 0/5